app/models/concerns/start_date_valid.rb

```diff
@@ -0,0 +1,16 @@
+module StartDateValid
+  extend ActiveSupport::Concern
+  included do
+    def start_date_valid?(recruitment_year, course)
+      if valid_start_date_range(recruitment_year).cover?(course.start_date)
+        true
+      else
+        course.errors.add :start_date, "#{start_date.strftime('%b %Y')} is not in the #{recruitment_cycle.year} cycle"
```

Comment:
Should validation just be checking that `start_date.strftime('%b %Y')` is one of the options that we pass to frontend?
Otherwise we have start date logic in two places.

Reply:
An existing example:
https://github.com/DFE-Digital/manage-courses-backend/blob/df8ec83c6e1d2e2be63701a620bbd98d9a01842a/app/models/course.rb#L460

Reply:
Good point. This logic pre-existed the additional work to expose the list via the metadata. I'll refactor it to check validate against the list sent.

Reply:
And done! Thanks Paul.